src/app/right-bar-creators/right-bar-creators-leaderboard/right-bar-creators-leaderboard.component.html

```diff
@@ -1,26 +1,26 @@
 <a
-  *ngFor="let profileEntryResponse of globalVars.rightBarLeaderboard"
-  [routerLink]="['/' + globalVars.RouteNames.USER_PREFIX, profileEntryResponse.Username]"
+  *ngFor="let profileEntryResponse of getActiveTabLeaderboard()"
```

Comment:
this was stored in GlobalVars so that we don't refetch top creators whenever the user navigates to a new page. also, a variable is better than a method because whenever angular decides to re-render the page the result is cached